cloudbuild.yaml

```diff
@@ -0,0 +1,27 @@
+# K8s infra build example: https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md
+# GCloud build docs: https://cloud.google.com/cloud-build/docs/build-config
+# Builds go to https://console.cloud.google.com/gcr/images/k8s-staging-sig-storage/GLOBAL
+# Build logs in https://testgrid.k8s.io/sig-storage-image-build
+timeout: 3600s
+options:
+  substitution_option: 'ALLOW_LOOSE'
+  machineType: 'E2_HIGHCPU_8'
+substitutions:
+  # K8s provides custom substitutions _GIT_TAG and _PULL_BASE_REF:
+  #   https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md#custom-substitutions
+  _GIT_TAG: '12345'  # e.g., vYYYYMMDD-hash, vYYYYMMDD-tag, or vYYYYMMDD-tag-n-ghash
```

Comment:
Maybe just leave it as an empty string?

Reply:
This is copy-pasted from the previous cloudbuild files. I don't know exactly why, but IIRC it was in the example file as well, so it seemed like it might have some use for testing/validation.